msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -47,11 +47,17 @@
 
 /**
  * <p>
- * This is the entry point for developer to create the public native applications and make API call to acquire tokens. MSAL {@link PublicClientApplication} provides three constructors allowing the client id to be set either via AndroidManifest.xml metadata or using constructor parameters.
+ * This is the entry point for developers to create the public native application and make API calls to acquire tokens.
+ * <p><b>Client ID:</b>  The clientID of your application is a unique identifier which can be obtained from the app portal.</p>
+ * <p><b>Authority:</b> A URL indicating a directory that MSAL can use to obtain tokens. In Azure AD
```

Comment:
But this is a public API doc, we should only document what we have in the API. We simply take in a string, converting to a URL is only implementation detail to the library itself.